resume.md

```diff
@@ -1,49 +1,49 @@
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Java, Python, JavaScript, Go, PostgreSQL, LaTeX
+* **Libraries/Frameworks**: Pandas, TensorFlow, React
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Incoming Software Engineering Intern**: Optiver | Chicago, IL *Jun 2021 - Aug 2021*
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
+**Incoming Software Engineering Intern**: Postmates X | San Francisco, CA *Mar 2021 - May 2021*
+
+**Software Engineering Intern**: NASA Jet Propulsion Laboratory | Pasadena, CA *Sep 2020 - Present*
+* Developing a vehicle simulation framework on the Mission Control team to scale support to dozens of robotic vehicles
+
+**Software Engineering Intern**: Google | Mountain View, CA *May 2020 - Aug 2020*
+* Increased Google Cloud query and data generation speed 10x by implementing a Markov chain-based tool in Java
+* Designed and scoped a project with open-ended specs by surveying engineers and iterating on design documentation
+* Generalized tokenization in several SQL dialects to increase new feature integration efficiency by 50%
+
+**Chief Operating Officer**: Illumna | Fort Lee, NJ *Oct 2017 - Oct 2019*
+* Led a team of 40+ college students to launch a nonprofit dedicated to mentoring under-resourced students
+* Increased 100+ mentees’ soft skills confidence by an average of 50% by writing 70+ pages of novel curriculum
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**EasyGov**: *Goverment forms made easy* | easygov.app *May 2020 - Present*
+* Developed a platform improving access to translated government forms with 1000+ form downloads in one month
```

Comment:
Can you support this idea with the technical implementation details?  What  tools did you use?  What are the next steps resulting from your partnerships?